requirements.txt

```diff
@@ -8,7 +8,9 @@ matplotlib >= 2.2, < 3.0; python_version >= '2.7' and python_version < '3.0'
 matplotlib >= 2.2, < 3.1; python_version >= '3.5' and sys_platform == 'win32'
 matplotlib >= 2.2; python_version >= '3.5' and sys_platform != 'win32'
 networkx
-numba
+llvmlite >= 0.31, < 0.32; python_version < '3.6'
```

Comment:
If this is for py27 support, then the constraint should be `<=2.7` instead of 3.6 ( i assume you rely on the fact that we are not building any 3.x lower than 3.6 but that makes the requirement semantically incorrect.

Reply:
Do we really need to put this llvm requirement here? Smells fishy

Reply:
seems weird that we all of a sudden need it at all indeed...

Reply:
I used 3.6 because the referenced version of numba is the last to support anything below 3.6.

Without the llvmlight requirement: pip will install `numba==0.47.0` if on python 2. The `requirements.txt` for `numba==0.47.0` specifies `llvmlite>=0.31` so pip will install `llvmlite==0.32`. `llvmlite==0.32` has no wheel for py27 on pypi and build fails (tested on travis/win and my linux installation).

Reply:
The alternative would be to ask the numba devs to release `numba==0.47.1` with a stricter version specifier for llvmlite.

Reply:
none of this makes sense. not son long ago everything worked fine for py27 so there must be a version of `numba` that works without specifying anything else...

Reply:
@gonzalocasas remind me again why we need a py27 build. can we not just do the transpile thing and check for py27 syntax?

Reply:
llvmlite=0.32 was released 7 hours ago and is shipped without a wheel for py27 because they dropped support for py27. That is what has changed.

Maybe it can still be built from sdist on py27 with some tinkering, but I haven't tried.

Reply:
@brgcode you are right, we don't really need a py27 built at all. Ideally we have the transpiler in place, but considering now syntax should all be py27 compatible anyway, and library dependencies do not affect IronPython stuff, the immediate thing to do would be to add IronPython to the travis build, so that we test it automatically.

I don't know how good the coverage is thou. @tetov you added/were adding/were thinking about adding coverage reports, right?

Reply:
@gonzalocasas I've only added coverage reports to compas_fab. I can set it up for compas too. 

Do you mean executing pytest with IronPython without installating the package?

Reply:
i have the feeling we have not yet really pin-pointed the problem.

in my opinion the goal is not to provide a working py27 build (py27 is history), but to make sure stuff will work in Rhino and GH. we are also not running any tests on `compas_rhino` and `compas_ghpython`.

therefore would it not make sense to include a "fake" py27 build just for the sake of Rhino/GH compatibility, potentially we a minimal requirements file?

@gonzalocasas ping